connector/rest/src/main/java/it/pagopa/selfcare/external_api/connector/rest/UserMsConnectorImpl.java

```diff
@@ -27,10 +29,11 @@ public UserMsConnectorImpl(UserControllerApi userControllerApi,
     }
 
     @Override
-    public List<UserInstitution> getUsersInstitutions(String userId) {
+    public List<UserInstitution> getUsersInstitutions(String userId, String institutionId, Integer page, Integer size, List<String> productRoles, List<String> products, List<PartyRole> roles, List<String> states) {
```

Comment:
Maybe we could wrap all these params into an object to make the method more readable.